src_assets/common/assets/web/config.html

```diff
@@ -818,6 +875,7 @@ <h2 class="accordion-header">
     "amd_vbaq": "enabled",
     "capture": "",
     "controller": "enabled",
+    "high_resolution_scrolling": "enabled",
     "install_steam_audio_drivers": "enabled",
     "ds4_back_as_touchpad_click": "enabled",
```

Comment:
Since you are touching this, can you alphabetize these keys?

Reply:
Yep